.github/workflows/bootstrap.yaml

```diff
@@ -0,0 +1,123 @@
+---
+name: Test bootstrapping a plugin
+on: pull_request
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: pulp_catdog
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: plugin_template
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
+      - name: Bootstrap catdog plugin
+        working-directory: plugin_template
+        run: |
+          .ci/bootstrap_catdog.sh
+
+  # Below this line we include the steps of the ci workflow of the generated plugin
+
+        # dev_requirements contains tools needed for flake8, etc.
+      - name: Install requirements
+        run: pip3 install -r dev_requirements.txt
+
+      # This check tries to lookup nonexisting commits
+      # - name: Check commit message
+      #   if: github.event_name == 'pull_request'
+      #   env:
+      #     GITHUB_CONTEXT: ${{ github.event.pull_request.commits_url }}
+      #   run: sh .github/workflows/scripts/check_commit.sh
```

Comment:
since we already check commits in another workflow, I think we can remove it

Reply:
unless we bring the other workflow to here https://github.com/pulp/plugin_template/blob/master/.github/workflows/check_commits.yaml

Reply:
Yes, i want to keep it here as a way of documentation.
Whenever we change the workflow template we need to adjust this file too.
And IMHO this comment can save a lot debugging time.